### ISSUE TITLE
Register Exiv2 with OpenSourceHub and CodeSee

### ISSUE THREAD

**Issue by clanmills** [Exiv2/exiv2#2436]
Folks

I recently received this email from Abe Esguerra abraham@codesee.io

_Hi Robin_

_Thank you for all the work you do and the efforts you put into Exiv2._

_I'm reaching out because I'm excited to invite you to join an open source community I've been helping build at Open Source Hub (https://opensourcehub.io), in just a few short months, we've grown to 10K+ developers, technologists and open source enthusiasts across Discord, TikTok and Substack._

_I'm passionate about connecting with other developers and that's the focus of what we do at OSH. We have a website where maintainers can add their projects for exposure and where contributors can browse through to find interesting projects they can contribute to. To add to that, we have a space over on Discord (discord.gg/opensource) where we can connect with each other in real-time for sharing memes, discussing projects and issues and making friends. What’s more, having your project verified thru out Verified Maintainer Program will give you exclusive access to amazing perks like distinguished promotions and many more! Check out https://opensourcehub.io/verified to know more!_

_I hope to see you in our community!_

I thanked Abe for this kind words.  I registered Exiv2:

https://opensourcehub.io/exiv2/exiv2

To be "verified",  requires a code map using CodeSee:  https://www.codesee.io/  However discord refused, _after several attempts_ to recognise my 2-factor code.  So I emailed Abe and asked him to contact Team Exiv2 on Github or the chat server.  I'm endorsing neither opensourcehub.io nor codesee.io.  I expect both have value and will involve effort from Team Exiv2.  Nothing is free. 

Happy Holidays and Best Wishes to you all for 2023.  Alison and I are in good spirits.  I've joined the local brass band.  Life is good.

![Odiham](https://user-images.githubusercontent.com/529982/208390512-7ffe416c-ea60-402e-9d2b-18f6b4d40721.jpg)

**Comment by kevinbackhouse** [comment]
@clanmills: I see this error message in the Codesee action:

```
===============================
Error accessing your API Token.
Please make sure the CODESEE_ARCH_DIAG_API_TOKEN is set correctly in your *repository* secrets (not environment secrets).
If you need a new API Token, please go to app.codesee.io/maps and create a new map.
This will generate a new token for you.
===============================
```

My guess is that the token is incorrectly named here: https://github.com/Exiv2/exiv2/settings/secrets/actions. Can you try changing the name from `CODESEE` to `CODESEE_ARCH_DIAG_API_TOKEN`? By the way, where did you get the token from? (In case somebody else needs to get a new token in the future.)

**Comment by clanmills** [comment]
I can logon to codesee using my GitHub credentials.  I've generated a new CODESEE_ARCH_DIAG_API_TOKEN key, added it to the secrets and deleted the "CODESEE" secret.

I'm interested to see what CodeSee does as it is probably useful.  I'm disappointed by the lack of help from the people who invited Exiv2 to register.

**Comment by clanmills** [comment]
A two-sentence chat with Shanea and we're running.  Here's the map:
https://app.codesee.io/maps/public/de096700-98c9-11ed-994f-5bc8c42801bc

Let's leave this issue open for a while so can add comments about my experience with the map.

**Comment by clanmills** [comment]
To be more than a code browser, l need a "Business" or "Enterprise" account with CodeSee.  I have requested a six-month free account for Team Exiv2 to learn more about CodeSee.

**Comment by clanmills** [comment]
This looks promising. CodeSee has:
1.  Upgraded my account to "Enterprise" (at no cost)
2. Been very responsive to instant messages.

**Comment by clanmills** [comment]
I'm going to close this issue because I still haven't understood what CodeSee does.  I thank the folks at CodeSee for their encouragement and assistance.

**Comment by kevinbackhouse** [comment]
> A two-sentence chat with Shanea and we're running. Here's the map: https://app.codesee.io/maps/d332f270-99c2-11ed-bcbf-97e003965916

I tried that link, but it took me to a page where I needed to register for an account. Then it asked me to authorize read/write access on my account, which I'm not keen to do. So I agree that we should pass on this.

![Screenshot from 2023-01-24 11-26-19](https://user-images.githubusercontent.com/4358136/214281367-238281b6-b02a-40c7-81ab-7af5846c0f54.png)
